xml/System.Net.Http.Headers/HttpHeaders.xml

```diff
@@ -113,6 +113,11 @@
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.ArgumentException">The name cannot be null or empty.</exception>
+        <exception cref="T:System.ArgumentNullException">The values cannot be null or empty.</exception>
+        <exception cref="T:System.InvalidOperationException">Misused header name. Make sure request headers are used with HttpRequestMessage, response headers with HttpResponseMessage, and content headers with HttpContent objects.</exception>
+        <exception cref="T:System.FormatException">The header name format is invalid.</exception>
```

Comment:
Typically, an exception type is listed once, with multiple exception conditions. My suggested correction for line 119 will do that, and line 120 should be deleted.